docs/cloud/visualize/interact-new-charts.md

```diff
@@ -37,6 +37,65 @@ With a quick glance you have immediate information available at your disposal:
 - [Chart area](#hover-over-the-chart)
 - [Legend with dimensions](#dimensions-bar)
 
+## Fundemental elements
+
+While Netdata's charts require no configuration and are easy to interact with, they have a lot of underlying complexity. To meaningfully organize charts out of the box based on what's happening in your nodes, Netdata uses the concepts of [dimensions](#dimensions), [contexts](#contexts), and [families](#families).
+
+Understanding how these work will help you more easily navigate the dashboard,
+[write new alerts](https://github.com/netdata/netdata/blob/master/health/REFERENCE.md), or play around
+with the [API](https://github.com/netdata/netdata/blob/master/web/api/README.md).
+
+### Dimensions
```

Comment:
Should this be 2 `#`, or the other sections 3 `#`?

Reply:
the latter, fixing, nice catch